### PR TITLE
fixes #154: chore: deploy scripts should read private key from env var

### LINKED ISSUE #154: chore: deploy scripts should read private key from env var
**What is the feature you would like to implement?**
Deploy scripts should read the private key from environment variable

**Why is this feature important?**
Simplifies the number of parameters that need to be passed in

**Will the protocol spec need to be updated??**
No

**How should this feature be built? (optional)**
```
1. `export PRIVATE_KEY=<key>` in `.env`
2. `uint256 privKey = vm.envUint("PRIVATE_KEY")`
3. `address deployer = vm.rememberKey(privKey)`
4. Deploy with `vm.broadcast(deployer)`
```
See: https://typefully.com/msolomon44/GtoP4UV

### PR DESCRIPTION
**Pull request by appleseed-iii**
## Motivation

Fix per #154

### CODE REVIEW

File: script/BundleRegistry.s.sol [hunk 1]
```diff
@@ -15,13 +15,11 @@ contract BundleRegistryScript is Script {
     address constant GOERLI_FORWARDER = address(0x7A95fA73250dc53556d264522150A940d4C50238);
     bytes32 constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
 
-    // TODO: Always update this to the address of the private key used to deploy the contracts on the network
-    address constant DEPLOYER = address(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266); // anvil deployer
+    address DEPLOYER = vm.addr(vm.envUint("PRIVATE_KEY"));
```
Comment (varunsrin) [inline script/BundleRegistry.s.sol:18]: if these aren't compile time constants anymore, we should lower case them

**Review by varunsrin — CHANGES_REQUESTED** [review]
Looks great! 

I had one small comment and you'll need to sign your commits before Github will allow a merge in this repo: https://github.com/farcasterxyz/hubble/blob/main/CONTRIBUTING.md#22-signing-commits

**Comment by appleseed-iii** [comment]
hey @varunsrin thanks. I force pushed my history to add the GPG signed commits. I also addressed your feedback in https://github.com/farcasterxyz/contracts/pull/193/commits/f36b4c2c5b3238c357549ad5625fb8003ff4b785

**Comment by varunsrin** [comment]
thank you! whats your farcaster username?

**Review by varunsrin — APPROVED** [review]

**Pull request merged**